src/main.cpp

```diff
@@ -54,7 +55,7 @@ int main(int argc, char *argv[]) {
   a->setOrganizationDomain("ptweb.me");
   a->setApplicationName("ptcollab");
   a->setApplicationVersion(Settings::Version::string());
-  QTranslator translator;
+  QTranslator translator, /*qtTranslator,*/ baseTranslator;
```

Comment:
(nit: Seems like this stray comment is unnecessary?)

Reply:
Look below